### ISSUE TITLE
Understanding Prerequisites for Test Rules

### ISSUE THREAD

**Issue by lwolberg** [w3c/wcag-act#511]
Background: A discussion on a recent call reiterated the following position regarding valid HTML: While valid HTML is desirable, valid HTML is not a success criteria in any WCAG, and so is not a pre-requisite for running a Test Rule. The group on the call felt that this pre-requisite (or lack therefof) would be informative to future users of the Test Rules. Shadi pointed out that the "Understanding" pages [https://www.w3.org/WAI/WCAG22/Understanding/understanding-act-rules] are a good place to share information like this. The committee suggested that we propose an addition to the "Understandings"

Proposed: We draft and add to the relevant "Understanding" page a heading where we can communicate this and perhaps other relevant pre-requisites (or lack thereof). 

To get things moving here is a strawman:

Test Rules Prerequisites

The HTML web page is available to the testing process. While it is desirable that the HTML be well formed (and pass, for example, the W3C markup validation service [https://validator.w3.org/], Test Rules (like most browsers) do not require valid HTML. 

If you'd like to contact me, please e-mail me at lionel@userway.org.

**Comment by nitedog** [comment]
How about adding a briefer note to the section "Test Rules are Partial Checks"?

Something like:
`Test Rules do not check for HTML validity beyond what is required for accessibility.`
